main.tf

```diff
@@ -77,7 +79,8 @@ resource "aws_s3_bucket_logging" "default" {
 resource "aws_s3_bucket_server_side_encryption_configuration" "default" {
   count = local.enabled ? 1 : 0
 
-  bucket = local.bucket_id
+  bucket                = local.bucket_id
+  expected_bucket_owner = var.expected_bucket_owner
```

Comment:
Is it possible that these various usages of `expected_bucket_owner` would differ? As in is it ever possible that we'd want a different bucket owner value for the `aws_s3_bucket_server_side_encryption_configuration` resource would be different than the `aws_s3_bucket_versioning` resource? I'm thinking no, but the question should be asked.

Reply:
(co-worker of @houserx-ioannis here!) - Good question - as far as I know this would never be different.